script/Ciel/Day 8/08_01_CIEL08_2_1.txt

```diff
@@ -1,27 +1,29 @@
 [sha:7cc854c20b617d1112edc84dfdacd9a2394486fa]{
 -- Page 10007, Offset 18704.
 -- ……怪しい。あの購買部好き……というか特定のパン好き……の先輩が、わざわざ外にランチしに行くとは思えない。
-...Very suspicious. I don't think that my school store loving, or rather special bread loving, Senpai would deliberately go out to have lunch.
+// god save me
+...Suspicious. My cafeteria loving Senpai, or rather, my specifically one kind of bread loving Senpai wouldn't go out of her way to have lunch somewhere outside school.
```

Comment:
...Suspicious. My cafeteria-loving Senpai—well, %{i}bread%{/i}-loving Senpai—wouldn't go out of her way to eat lunch outside